### ISSUE TITLE
Firefox add-on out-of-date?

### ISSUE THREAD

**Issue by timeimp** [reduxjs/redux-devtools#1493]
Hey Redux team,

I was trying to workout why Redux tools was randomly freezing and have noted that the [official Firefox add-on page](https://addons.mozilla.org/en-US/firefox/addon/reduxdevtools/) has not been updated in 6 months, but the Edge store version has.

Is there a different way we should be updating the Firefox version (manually or otherwise)?

[Edge Add-on store page](https://microsoftedge.microsoft.com/addons/detail/redux-devtools/nnkgneoiohoecpdiaponcejilbhhikei) - v3.1.1, updated August 2023
[Firefox add-on page](https://addons.mozilla.org/en-US/firefox/addon/reduxdevtools/) - v3.0.19, updated March 2023

**Comment by Methuselah96** [comment]
Hey @timeimp!

Thanks for the issue. You happened to check the versions in the midst of us trying to work through some bugs. We disabled v3.1.1 in Firefox due to https://github.com/reduxjs/redux-devtools/issues/1480. We fixed that in v3.1.2, but uncovered some other bugs as well after updating it in the Chrome Web Store. v3.1.3 should be going out to all browsers today, although sometimes it can take a few days to get approved by each browser's extension team after we submit a new version.

In general, we release to Chrome first using a staged rollout, since it's the only browser that offers that feature. Once everything looks in Chrome we release it Firefox and Edge, which is often why the versions might be out-of-sync for a time.
